common/types.go

```diff
@@ -33,6 +33,9 @@ import (
 	"github.com/grafana/xk6-browser/api"
 )
 
+// RegisterCallbackFn represents the function used to register an event loop callback.
+type RegisterCallbackFn func() func(func() error)
+
```

Comment:
`RegisterCallbackFunc`: I didn't want to bring this up but can we at least follow the idiomatic Go naming conventions here? For example: https://pkg.go.dev/bufio@go1.17.8#SplitFunc 🙏

Reply:
I find it funny that Go is usually in favor of terseness, but the convention is `Func` instead of the widely known `Fn`. :man_shrugging:

In any case, I'll probably remove this [once I attach `VU` to the context](https://github.com/grafana/xk6-browser/pull/268#discussion_r822537491).

Reply:
Resolved by e5076d5.